src/imitation/util/networks.py

```diff
@@ -1,150 +1,32 @@
 """Helper methods to build and run neural networks."""
 
 import collections
-import contextlib
-from typing import Callable, Dict, Iterable, Optional, Sequence, Tuple
+from typing import Callable, Iterable, Optional
 
-import gym
-import tensorflow as tf
-from stable_baselines.common.input import observation_input
-
-LayersDict = Dict[str, tf.layers.Layer]
```

Comment:
If this is what was causing all the test failures, can we just add it back in and remove it in Sam's PR? This seems cleaner to me than merging Sam's PR into this branch. (But there may well be other failures that would still happen even if we added this back in.)

Reply:
I expect there to be other failures because changing all the policies to Torch will break the adversarial algorithms.